core/templates/base.html

```diff
@@ -42,16 +42,31 @@
         </div>
         <div class="navbar-collapse collapse" id="navbar-main">
             <ul class="nav navbar-nav navbar-right">
-                <li><a href="{% url 'social:begin' 'yammer' %}?next={{ request.path }}">Login</a></li>
+                <li><a href="{% url 'social:begin' 'yammer' %}?next={{ request.path }}">*</a></li>
```

Comment:
Not sure what the point of the asterisk is, or even the login URL when we're already logged in...  If anything it'd make sense to have a logout in the top right, with a check to only display the link if you're actually logged in.  Thoughts?